README.md

```diff
@@ -8,8 +8,34 @@ Level Up is a Steam game recommendations site where users can submit their own a
 
 ## Run it yourself
 
-Level Up's server components are written in Go and operate on top of an Amazon DynamoDB database (DynamoDB's free tier is more than enough to host).
+### 1. using Docker
+
+Make sure you have the .env file if not run: `cp .env.example .env && $EDITOR .env`
+```
+`docker build -t levelup .` 
+
+`docker run --env-file .env -p 4000:4000 levelup` 
+```
+
+or
+
+```
+`docker run --env-file .env -p 4000:4000 bikerwhocodes/levelup` 
+```
 
+or
+
+```
+`docker-compose up`
+
+or 
+
+`docker-compose up -d` for headless mode
```

Comment:
This probably should be under its own heading? I'll leave it to you on this one, but "for headless mode" should probably be outside the code block.